swarm/tools/flow_studio_ui/fragments/60-modals.html

```
@@ -67,6 +67,7 @@ <h3 id="shortcuts-modal-title">Keyboard Shortcuts</h3>
 <div id="run-detail-modal" class="selftest-modal" role="dialog" aria-modal="true" aria-labelledby="run-detail-modal-title" data-uiid="flow_studio.modal.run_detail">
   <div class="selftest-step-content" data-uiid="flow_studio.modal.run_detail.body">
     <button class="selftest-modal-close" id="run-detail-close" aria-label="Close run detail modal" data-uiid="flow_studio.modal.run_detail.close">×</button>
+    <button id="run-detail-rerun-btn" onclick="rerunDetail()" data-uiid="flow_studio.modal.run_detail.rerun" style="display: none;">Re-run</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)